test/JDBC/expected/pivot-vu-verify.out

```diff
@@ -738,6 +738,8 @@ PIVOT (
 ) AS pvt;
 SELECT TOP 10 * FROM pivot_insert_into ORDER by 1, 2;
 GO
+~~ROW COUNT: 197~~
```

Comment:
wouldn't `TOP 10` return only 10 rows?

Reply:
This one : 
```
INSERT INTO pivot_insert_into
SELECT  ManufactureID, EmployeeID, [2] AS STORE2, [3] AS STORE3, [4] AS STORE4, [5] AS STORE5, [6] AS STORE6
FROM
(
    SELECT ManufactureID, EmployeeID, ItemID, StoreID
    FROM StoreReceipt
)AS srctable
PIVOT (
    COUNT (ItemID)
    FOR StoreID IN ([2], [3], [4], [5], [6])
) AS pvt;

```

Reply:
The TOP 10 is another statement

Reply:
> wouldn't `TOP 10` return only 10 rows?

There's a long-standing issue with SELECT-INTO that  it does not report the row count (though @@rowcount is correct). See [BABEL-540](https://jira.rds.a2z.com/browse/BABEL-540)